projects/plugins/jetpack/modules/subscriptions/subscribe-modal/class-jetpack-subscribe-modal.php

```
@@ -228,43 +215,11 @@ public function has_subscription_cookie() {
 	}
 }
 
-add_filter(
-	'jetpack_subscriptions_modal_enabled',
-	array(
-		'Jetpack_Subscribe_Modal',
-		'should_load_subscriber_modal',
-	)
-);
-
-/**
- * Filter for enabling or disabling the Jetpack Subscribe Modal
- * feature. We use this filter here and in several other places
- * to conditionally load options and functionality related to
- * this feature.
- *
- * @since 12.4
- *
- * @param bool Defaults to false.
- */
-if ( apply_filters( 'jetpack_subscriptions_modal_enabled', false ) ) {
```

We could keep the filter but default it to `true` in case site administrators want to disable/hide this feature?.

---

Do we do that for other features? Feels like something we could add if there's demand, rather than leaving it in-case?